index.html

```diff
@@ -0,0 +1,690 @@
+            <nav id="navbar">
+                <div id="nav-icons">
+                    <div id="navbar-flex-container">
+                        <div id="tnw-logo-background">
+                            <img id="nav-tnw-logo" alt="The Next Web" src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152646im_/https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg">
+                        </div>
+                        <div id="navbar-links">
+                            <div id="top-nav-links">
+                                <div id="top-nav-links-left">
+                                    <div class="inline-list" id="news-link"><a href="#" class="normal-link ">News</a></div>
+                                    <div class="inline-list"><a href="#" class="normal-link">Events <i class="fa fa-caret-down" aria-hidden="true"></i></a></div>
+                                    <div class="inline-list"><a href="#" class="normal-link">Business <i class="fa fa-caret-down" aria-hidden="true"></i></a></div>
+                                    <div class="inline-list"><a href="#" class="normal-link">AMAs</a></div>
+                                    <div class="inline-list"><a href="#" class="normal-link">Spaces</a></div>
+                                    <div class="inline-list"><a href="#" class="normal-link">Terms &amp; Conditions</a></div>
+                                </div>
+                                <div id="top-nav-links-right">
+                                    <div class="inline-list"><a href="#" class="normal-link">About</a></div>
+                                    <div class="inline-list"><a href="#" class="normal-link">Advertise</a></div>
+                                    <div class="inline-list"><a href="#" class="normal-link">Jobs</a></div>
+                                    <div class="inline-list"><a href="#" class="normal-link">Contact</a></div>
+                                </div>
+                            </div>
+                            <div id="bot-nav-links">
+                                <div class="bot-nav-text-links">
+                                    <div class="bot-nav-link"><a href="#" class="normal-link">Latest</a></div>
+                                    <div class="bot-nav-link"><a href="#" class="normal-link">Hard fork</a></div>
+                                    <div class="bot-nav-link"><a href="#" class="normal-link">Plugged</a></div>
+                                    <div class="bot-nav-link"><a href="#" class="normal-link">Readme</a></div>
+                                    <div class="bot-nav-link"><a href="#" class="normal-link">Growth quarters</a></div>
+                                    <div class="bot-nav-link"><a href="#" class="normal-link over-1215">Shift</a></div>
+                                    <div class="bot-nav-link"><a href="#" class="normal-link over-1215">Neural</a></div>
+                                </div>
```

Comment:
- [ ] **[Optional]** Your project does not have a medium: 768px breakpoint, which could help to fix the small mess you have here.

Original at 781px 👇 
![image](https://user-images.githubusercontent.com/64360974/113470083-71f8d600-945b-11eb-8ce6-72a2b32e3364.png)

yours at 781px 👇 
![image](https://user-images.githubusercontent.com/64360974/113470097-83da7900-945b-11eb-9e17-d3c147d8a88f.png)

Reply:
Noted and thank you for your kind response! will fix the bug. 👍